src/main/java/be/acara/frontend/controller/EventController.java

```diff
@@ -49,15 +49,16 @@ public String displayEvent(@PathVariable("id") Long id, ModelMap model) {
     public String findAllEvents(ModelMap model,
                                 @RequestParam(name = "page", defaultValue = "1", required = false) int page,
                                 @RequestParam(name = "size", defaultValue = "20", required = false) int size) {
-        EventDtoList eventList = eventService.findAllEvents(page - 1, size);
-        int totalPages = eventList.getTotalPages();
+        addCategories(model);
```

Comment:
It is not required for the display of all events to have the categories provided to them.

Reply:
Remove this one.